Tests/ufoLib/glifLib_test.py

```diff
@@ -285,6 +287,25 @@ def test_read_ensure_x_y(self):
 		with pytest.raises(GlifLibError, match="Required y attribute"):
 			readGlyphFromString(s, _Glyph(), pen, validate=False)
 
+	def test_parse_xml_note_cdata(self):
```

Comment:
@anthrotype May not be quite how you would have done it, sorry, but it works well enough. :sweat_smile: